fastapi_filter/contrib/sqlalchemy/filter.py

```diff
@@ -39,6 +39,7 @@ def _backward_compatible_value_for_like_and_ilike(value: str):
     "isnull": lambda value: ("is_", None) if value is True else ("is_not", None),
     "lt": lambda value: ("__lt__", value),
     "lte": lambda value: ("__le__", value),
+    "between": lambda value: ("between", (value[0], value[1])),
```

Comment:
It would be beneficial to enhance protection against `IndexError`. For instance, if a client passes `__between=21`, instead of `between=21,33`.